MessageBoardTests/MessageThreadTests.swift

```diff
@@ -10,6 +10,23 @@ import XCTest
 @testable import Message_Board
 
 class MessageThreadTests: XCTestCase {
+    func testCreateMessageOnServer() {
+        let messageController = MessageThreadController()
+        let thread = MessageThread(title: "Test")
+        
+        messageController.createMessageThread(with: thread.title) {
+            XCTAssertTrue(!messageController.messageThreads.isEmpty)
+        }
+    }
+    
+    func testFetchMessagesFromServer() {
+        let messageController = MessageThreadController()
+        
+        messageController.fetchMessageThreads {
+            XCTAssertTrue(!messageController.messageThreads.isEmpty)
+        }
+    }
+    
```

Comment:
These tests feel a bit sparse. Test the JSON decoding/encoding or something else that you fixed at the lower level (non-app logic).